libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Security/SFSDKCryptoUtils.h

```diff
@@ -159,15 +159,15 @@ extern NSUInteger const kSFPBKDFDefaultSaltByteLength;
  * @param keyRef Keyref used in encryption
  * @return `NSData` object containing the encrypted Data, or `nil` if encryption failed.
  */
-+ (nullable NSData*)encryptUsingRSAforData:(NSData *)data withKeyRef:(SecKeyRef)keyRef SFSDK_DEPRECATED(11.1.1, 12.0, "Will be removed.");
```

Comment:
What is the replacement method?

Reply:
[these ones](https://github.com/forcedotcom/SalesforceMobileSDK-iOS/blob/dev/libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Security/CryptoUtils.swift)  (used [here](https://github.com/forcedotcom/SalesforceMobileSDK-iOS/blob/a91b87fc346bd09f6a6582fef3fe7b10a63c147c/libs/SalesforceSDKCore/SalesforceSDKCore/Classes/PushNotification/SFSDKPushNotificationDecryption.m#L190) and [here](https://github.com/forcedotcom/SalesforceMobileSDK-iOS/blob/a91b87fc346bd09f6a6582fef3fe7b10a63c147c/libs/SalesforceSDKCore/SalesforceSDKCore/Classes/PushNotification/SFSDKPushNotificationDecryption.m#L193) for notifications)